setup.py

```diff
@@ -16,32 +16,43 @@
 # We use the version to construct the DOWNLOAD_URL.
 DOWNLOAD_URL = ''.join((REPO_URL, '/tarball/release/', VERSION))
 
-REQUIREMENTS = ['pystache', 'Sphinx','kruxstatsd', 'argparse', 'lockfile', 'future']
+REQUIREMENTS = ['pystache', 'kruxstatsd', 'lockfile', 'future']
```

Comment:
Why remove `argparse`?

Reply:
Isn't `argparse` already part of Python stdlib? I don't think you need to install it separately.

Reply:
oh, duh. yeah, that groks.